infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```diff
@@ -194,7 +194,7 @@ private void processor() {
                                 log.warn("Unknown BatchWriterOperation {}", currOp);
                         }
                     } catch (Exception e) {
-                        log.error("Stream log error. Batch [queue size={}]. StreamLog: [trim mark: {}, tails: {}].",
+                        log.trace("Stream log error. Batch [queue size={}]. StreamLog: [trim mark: {}, tails: {}].",
```

Comment:
Apart from reducing the log level, can we add one more log line which does not prin the streamLog.getAllTails.
This will keep just one line for debugging.

Reply:
fixed